packages/cli/src/utils/utils.ts

```diff
@@ -113,27 +114,131 @@ export function findReplace(manifest: string, replacer: RegExp, value: string):
   return manifest.replace(replacer, value);
 }
 
+export function replaceArrayValueInTsManifest(manifest: string, key: string, newValue: string): string {
+  const start = manifest.indexOf(`${key}:`);
+  if (start === -1) return manifest; // If the value is not found, return the original manifest
```

Comment:
I think rather than this function doing a no-op it would be better to throw and let the caller decide what to do.

Same with the open brackets check